### DISCUSSION TITLE
2009/05/14/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E7%9A%84%E5%86%99%E4%BD%9C%E8%BF%87%E7%A8%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#260] [Announcements]
# 2009/05/14/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E7%9A%84%E5%86%99%E4%BD%9C%E8%BF%87%E7%A8%8B

写书事记：

https://blog.qizhen.xyz/2009/05/14/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E7%9A%84%E5%86%99%E4%BD%9C%E8%BF%87%E7%A8%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Yuan
2009年06月2日 04:50 编辑
好遗憾要到8月份。 家里人马上就要给我寄书了呢，看来是没有机会看到图书版的了。

回复
Qizhen
2009年05月21日 12:59 编辑
我没有QQ，MSN用的也比较少，你可以把 msngroup7773@hotmail.com 加进来和我交流。

回复
cslc
2009年05月19日 12:40 编辑
我也是想买这本，真的还要等这么久吗，有点迫不及待了呀！

回复
cslc
2009年05月19日 12:39 编辑
你好，前辈，我是个新手，想学习LV，不介意的话，能给个交流的QQ或者什么的，空间，联系的。

回复
Jucao
2009年05月15日 14:16 编辑
4年磨一剑，不容易！

回复
秋杰
2009年05月15日 10:28 编辑
我很高兴听到你出书！一定要快哦！我可是想买哦！

回复
